src/components/layout/Layout.test.js

```diff
@@ -1,41 +1,34 @@
-import { mount } from "enzyme";
-import React from "react";
-import Layout from "./Layout";
-import { StyledContent } from "./LayoutStyled";
-import Menu from "./Menu/Menu";
-import { BrowserRouter as Router } from "react-router-dom";
-import { ThemeProvider, createTheme } from "@mui/material";
+import React from 'react';
+import Layout from './Layout';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ThemeProvider } from '@mui/material';
+import { render, screen } from '@testing-library/react';
+import theme from '../UI/theme';
 
-const theme = createTheme({
-  zIndex: {
-    drawer: 1200,
-  },
-});
+describe('Layout component', () => {
 
-describe("Layout component", () => {
-  let wrapper;
+  const TestWrapper = (props) => {
```

Comment:
I like your idea with the Wrapper. Please, update all other tests using Enzyme: App.test.js, ClientRoutes.test.js, ErrorMessage.test.js, Loader.test.js, Login.test.js and remove it from setupTests.js